app/views/students/student_report.pdf.erb

```diff
@@ -124,13 +124,13 @@
                 <span style="font-weight:bold;">
                   <%= format_date(service.date_started) %> -
                   <% if service.discontinued_at.present? %>
-                    <%= "#{format_date(service.try(:discontinued_at))}" %>
+                    <%= "#{format_date(service.discontinued_at)}" %>
```

Comment:
This is a 👍  change, there's already a `present?` call up top so `try` here is duplicative.